daemon/api.go

```diff
@@ -281,6 +281,11 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 		includeLocal = true
 	}
 
+	var includeTypes []string
+	if len(query["types"]) > 0 {
```

Comment:
In this situation [we only get the first value](https://play.golang.org/p/Mtb506cMDQ) it would seem :)